### DISCUSSION TITLE
Sleep state correct?

### DISCUSSION THREAD

**Discussion by KoljaHenckel** [teslamate-org/teslamate#2954] [Q&A]
Hi!
I had Teslamate running for 8 month until May, the I had troubles with my home assistant. I re-activated it last week. In the meantime I upgraded to the MCU2 ( I have a 2016 Model S).
I saw that the statistics in grafana show no sleep time any more:
 online (ein Tag, 58%, 29x) offline (14 Stunden, 30%, 16x) driving (3 Stunden, 7%, 10x) charging (3 Stunden, 5%, 2x)

compared to 40% at the beginning of the year:
 asleep (5 Tage, 42%, 174x) online (3 Tage, 24%, 225x) offline (3 Tage, 22%, 3x) charging (ein Tag, 8%, 18x) driving (9 Stunden, 3%, 31x)

Nevertheless, when I look at the status page of teslamate I see that the car seems to be sleeping:
Status | schläft ein seit 19 h, 1 min

Is this just an error in saving the state?

**Comment by ktsenvolt** [comment]
#2911

**Reply by KoljaHenckel** [reply]
Thanks for pointing out, but I have the newest iOS app version (whosoever) and the Tesla update 2022.36.2. I am not sure how this linkes issue should help..?
